src/framework/preferences/css/stylus/utils/Themes.styl

```diff
@@ -125,19 +151,45 @@ build-themes-Enactors(contrastThemes) {
                 border-left-color: fColor !important;
             }
 
+            if (selectedFColor && selectedBColor) {
+                &::selection,
+                ::selection {
+                    color: selectedFColor !important;
+                    background-color: selectedBColor !important;
+                }
+            }
+
+            if (disabledColor) {
```

Comment:
Minor thing, but I think I would swap the order here of the `if (selectedFColor...` and `if (disabledColor...` blocks so that they match the order of specification of the values above beginning line 136.